backend/tracim_backend/views/core_api/account_controller.py

```diff
@@ -50,10 +28,8 @@ def account_route_get(self, context, request: TracimRequest, hapic_data: HapicDa
         without giving directly user id.
         This route generate a HTTP 307 with the right url
         """
-        return redirect_account_route(
-            user_id=request.current_user.user_id,
-            path_suffix=request.matchdict.get("path_suffix"),
-            get_params=request.GET,
+        return HTTPTemporaryRedirect(
+            request.url.replace("/me", "/{user_id}".format(user_id=request.current_user.user_id), 1)
```

Comment:
Just converted the redirection from a "rebuild url" model to a more simpler approach with replace. @grignards @buxx @raphj which approach did you prefer here, and did you see any issue with this approach (possibly proxy related) ?

Reply:
You should probably replace the whole prefix, to avoid risky random unrelated replacements in the url, no?

Reply:
you are right, according to the code, as we add route with` /users/me{path_suffix:.*} ` (and the prefix for api is done in another place,
The best of both approach seems:

```
request.url.replace("/users/me", "/users/{user_id}".format(user_id=request.current_user.user_id), 1)
```
This way we don't mix thing that are not related (prefix for api for example), and we do maximize risk.

Reply:
Or even replace `/users/me/` to be sure (even if it is unlikely we'll use a route like `/users/meow` :wink:

Reply:
That's not really possible as `/users/me` endpoint exist.